src/views/TipsList.vue

```diff
@@ -38,12 +38,42 @@
               >
                 {{ $t('views.TipList.SortingHighestRated') }}
               </a>
+              <div
+                v-if="showMenu"
+                class="overlay-dots"
+                @click="showMenu = false"
+              />
+              <div class="actions-menu">
```

Comment:
This probably needs to be a component with a slot for the dropdown content

Reply:
There is an issue for unifying the dropdown styles. In order not to double the work I guess it could be optimized in #549. @mradkov What do you think?

Reply:
> There is an issue for unifying the dropdown styles. In order not to double the work I guess it could be optimized in #549. @mradkov What do you think?

Agree